app/services/flowmanagement/PageRouter.scala

```diff
@@ -30,6 +30,11 @@ trait PageRouter[A] {
                                                               ec: ExecutionContext
   ): Future[Result]
 
+  def getPageRouteNewAuth(credId: String, model: A, edit: Boolean = false)(implicit
+                                                    hc: HeaderCarrier,
+                                                    ec: ExecutionContext
+  ): Future[Result]
+
```

Comment:
new router should be renamed to getRoute only, all references to old router should be removed